R/spectraTab.R

```diff
@@ -1,42 +1,39 @@
 #' tuneParams tab showing size spectra
-#' 
+#'
 #' This tab shows:
-#' 
+#'
 #' * a plot of the biomass density as a function of log size,
-#' including all species and the resource. 
+#' including all species and the resource.
 #' * A slider for scaling the background (resource and possibly background
 #' species).
 #' * If there are background species, buttons to adjust or remove those
 #' background species.
-#' 
+#'
 #' @inheritParams abundanceControl
 #' @param logs Environment holding the log of steady states.
 #' @param trigger_update Reactive value used for triggering update of
 #'   species parameter sliders.
 #' @param ... Unused
 spectraTab <- function(input, output, session,
                        params, logs, trigger_update, ...) {
-    
+
     ## Plot spectra ####
     output$plotSpectra <- renderPlotly({
         # if (input$binning == "Logarithmic") {
-        power <- 2
+            power <- 2
```

Comment:
Why the extra indent?

Reply:
automatic indentation when I was testing plotSpectra, I removed it.